<launch>
  <node pkg="ndt_calibration" type="ndt_calib_online_node" name="ndt_calib_online_node" output="screen">
  	<param name="points_topic" value="" />
	<param name="laser_topic" value="/laserscan0" />
        <param name="ct" value="false"/>
	<param name="resolution" value="2"/>
        <param name="min_nb_calibration_pairs" value="10" />
        <param name="max_nb_calibration_pairs" value="50"/>
  </node>
  
</launch>
